packages/changesets-renovate/src/index.ts

```diff
@@ -97,7 +97,10 @@ export async function run() {
 
   console.log('Detected branch:', branch)
 
-  if (!branch.current.startsWith('renovate/')) {
+  if (
+    !branch.current.startsWith('renovate/') &&
+    !process.env['SKIP_BRANCH_CHECK']
```

Comment:
Maybe the idea behind https://github.com/scaleway/scaleway-lib/issues/1706 was to configure the name of the branch ? 

```ts
branch.current.startsWith('renovate/')
```
 per 
```
ranch.current.startsWith(BRANCH_NAME)
```

We can keep the SKIP_BRANCH_CHECK but it's will be more response to the issue your are tracking.

Reply:
Good point. I've added an option to have a custom branch name.